app/assets/src/components/common/MetadataUpload.jsx

```diff
@@ -57,6 +61,16 @@ class MetadataUpload extends React.Component {
       issues,
       validatingCSV
     });
+    if (validatingCSV) {
```

Comment:
IMO I don't think you need this first one.

This code is a bit confusing, but whenever the user validates metadata, this function is called twice.

The first time is before the request is sent. It just clears the errors and sets validatingCSV to true so <MetadataUpload> can display a loader.
The second time is after the request is returned. This function is called with the actual issues and sets validating CSV to false.

So when validatingCSV is true, issues.errors.length is always 0. It's just to reset the state.

Reply:
Thanks for explaining. I noticed the double logging and I didn't know what to do exactly. I will change it to only log when `validatingCSV` is false.